### PR TITLE
Task related to routing

### PR DESCRIPTION
**Pull request by Abhinav4952**
Task details:[-AngularTraining/user-management-staticdata/Homework 2.docx](https://github.com/Abhinav4952/EPAMDEPTraining/compare/usermanagementstaticdata?expand=1#diff-830e71cd39372a7ffad872c2616104ff)

### CODE REVIEW

File: AngularTraining/user-management-staticdata/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { F1c1Component } from './feature1/f1c1/f1c1.component';
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/app-routing.module.ts:4]: Use better naming convention for component names

File: AngularTraining/user-management-staticdata/src/app/feature1/deleted-users.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'deletedUsers'
+})
+export class DeletedUsersPipe implements PipeTransform {
+
+  transform(value: any[]): unknown {
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/feature1/deleted-users.pipe.ts:8]: change method return type

File: AngularTraining/user-management-staticdata/src/app/feature1/f1c1/f1c1.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  styleUrls: ['./f1c1.component.css']
+})
+export class F1c1Component implements OnInit {
+
+  userList = [];
+
+  constructor(private usersService: UsersService) { }
+
+  activateUser(userId) {
+    for (var i = 0; i < this.usersService.users.length; i++) {
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/feature1/f1c1/f1c1.component.ts:16]: use forEach

File: AngularTraining/user-management-staticdata/src/app/feature1/feature1.module.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { F1c1Component } from './f1c1/f1c1.component';
+
+import { DeletedUsersPipe } from './deleted-users.pipe';
+
+
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/feature1/feature1.module.ts:7]: format your document, there are many unwanted lines

File: AngularTraining/user-management-staticdata/src/app/feature2/active-users.pipe.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'activeUsers'
+})
+export class ActiveUsersPipe implements PipeTransform {
+
+  transform(value: any[]): unknown {
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/feature2/active-users.pipe.ts:8]: change method return type

File: AngularTraining/user-management-staticdata/src/app/login/login.component.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+  constructor(private usersService: UsersService,private router: Router) { }
+
+  userName = 'Hello';
+  usersList=[];
+  beUser=[];
+  ngOnInit(): void {
+    this.usersList=this.usersService.users;
+    this.usersService.cast.subscribe(userr=>this.beUser=userr);  
+    console.log(this.beUser);
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/login/login.component.ts:20]: remove console.log statement

File: AngularTraining/user-management-staticdata/src/app/users.service.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  providedIn: 'root'
+})
+export class UsersService {
+  currentUser = '';
+
+  userChanged: EventEmitter<string> = new EventEmitter();
+  userListChanged:EventEmitter<any> = new EventEmitter();
+  
+  users = [
+    { id: 1, name: 'User 1',lastName:'chal', login: 'user1', age: 25,password:'123',isDeleted:true },
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/users.service.ts:14]: try to store dummy data in different json file.

File: AngularTraining/user-management-staticdata/src/app/login/login.component.ts (hunk 6 above)
Comment (Abhinav4952) [reply AngularTraining/user-management-staticdata/src/app/login/login.component.ts:20]: Removed console.log statement

**Comment by Abhinav4952** [comment]
Resolving all req and raised a PR:-https://github.com/Abhinav4952/EPAMDEPTraining/pull/16

**Pull request closed without merging**